eureka-client/src/test/java/com/netflix/discovery/InstanceInfoReplicatorTest.java

```diff
@@ -76,7 +76,7 @@ public void testOnDemandUpdateRateLimiting() throws Throwable {
         assertTrue(replicator.onDemandUpdate());
         Thread.sleep(10);  // give some time for execution
         assertFalse(replicator.onDemandUpdate());
-        Thread.sleep(10);
+        Thread.sleep(1000);
```

Comment:
Was this flakey?

Reply:
Yeah. I'm guessing both those `onDemandUpdate()` calls didn't have time to finish so the rate limiter counts never triggered. Increasing the sleep give them time to finish and the counter to increment.